src/syncerd/bitcoin_syncer.rs

```diff
@@ -304,22 +329,11 @@ impl Synclet for BitcoinSyncer {
                 let new_block = rpc.new_block_check();
                 if let Some(block_notif) = new_block {
                     state.change_height(block_notif.height, block_notif.block_hash.to_vec());
-
-                    for (_, watched_tx) in state.transactions.clone().iter() {
-                        let tx_id: bitcoin::Txid =
-                            bitcoin::Txid::from_hex(&hex::encode(watched_tx.hash.clone())).unwrap();
-                        let tx = rpc.client.transaction_get_verbose(&tx_id).unwrap();
-
-                        let blockhash = match tx.blockhash {
-                            Some(bh) => Some(bh.to_vec()),
-                            None => none!(),
-                        };
-
-                        state.change_transaction(tx.txid.to_vec(), blockhash, tx.confirmations)
-                    }
                 }
-                trace!("pending events: {:?}", state.events);
-
+                if !state.events.is_empty() || i == 0 || i % 25 == 0 {
```

Comment:
Surely this is interim to get some more data through, right?

Reply:
the hack is just to make it run at i = 0, the 25 was just to get a trace log once in a while, to understand syncer is running